pkg/kubelet/kubelet.go

```diff
@@ -507,6 +507,15 @@ func NewMainKubelet(kubeCfg *componentconfig.KubeletConfiguration, kubeDeps *Kub
 	}
 	glog.Infof("Hairpin mode set to %q", hairpinMode)
 
+	// TODO(#36485) Remove this workaround once we fix the init-container issue.
+	// Touch iptables lock file, which will be shared among all containers.
+	f, err := os.OpenFile(utilipt.LockfilePath16x, os.O_CREATE, 0600)
+	if f == nil || err != nil {
```

Comment:
nit: don't think you'll need to check `f == nil` since you already checked `err != nil`